src/tests/keywords/unit_test_module.conf

```diff
@@ -35,6 +35,7 @@ modules {
 	}
 
 	csv {
+		key = "field1"
```

Comment:
The `key` here is similar to the `key` in rlm_files.  It's not a field name, it should be the same as the argument to the `map` e.g.

`key = "%{tolower:%{User-Name}}"`

Reply:
done